Source/Shaders/GlobeVS.glsl

```diff
@@ -21,6 +21,8 @@ varying vec3 v_positionEC;
 varying vec3 v_textureCoordinates;
 varying vec3 v_normalMC;
 varying vec3 v_normalEC;
+varying float v_slope;
```

Comment:
Everything in here should also be surrounded by the `APPLY_MATERIAL` define too.

Reply:
Fixed